src/Evolution/Executables/GeneralizedHarmonic/EvolveGhBinaryBlackHole.hpp

```diff
@@ -192,9 +192,10 @@ struct EvolutionMetavars {
         ::ah::vars_to_interpolate_to_target<volume_dim, ::Frame::Grid>,
         ::ah::vars_to_interpolate_to_target<volume_dim, ::Frame::Inertial>>;
     using compute_vars_to_interpolate = ah::ComputeHorizonVolumeQuantities;
-    using tags_to_observe = ::ah::tags_for_observing;
+    using tags_to_observe = ::ah::tags_for_observing<Frame::Grid>;
```

Comment:
You have to change this in the GHWithHorizon executable as well since it uses the same type alias